### ISSUE TITLE
Activity indicator provided sometimes appear after a delay or after cellForRowAtIndexPath is called again.

### ISSUE THREAD

**Issue by ankit-betterbutter** [rs/SDWebImage#1774]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / tvos
 Platform Version        | e.g. 8.0
 SDWebImage Version      | e.g. 3.7.6
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 7.3
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by mythodeia** [comment]
can you replicate this in the demo project? does this happen in the latest version of the library?

**Comment by bpoplauschi** [comment]
@ankit-betterbutter the question was if this scenario happens WITH OUR DEMO, not to create a demo.

**Comment by ankit-betterbutter** [comment]
for anyone facing similar issue -

The mistake i was doing was setting the showActivityIndicator after the webservice call.

Kindly follow this while adding activity indicator

    cell.imageView.setShowActivityIndicator(true)
    cell.imageView.setIndicatorStyle(.gray)
    cell.imageView?.sd_setImage(with: url)
